### ISSUE TITLE
Please teach me how to use the Free Manga Downloader program properly

### ISSUE THREAD

**Issue by amateurdota** [riderkick/FMD#646]
I want to know this because it seems this program is really different compared to other old/dead manga downloaders such as domdomsoft which are so extremely easy to use, as when I tried to use it as it is the manga chapters never show at all and I truly have no idea how to add the other manga websites into manga info tab. So that means there's something else needed to be done for running the program properly as far as I know. And so please teach me in simpler terms about what to do to use this program, I run updater and it simply never ever work not even in a month, so I beg you teach me how to use the program please.

**Comment by kavin-90** [comment]
you can just copy website url into manga info page and it will show chapter list

**Comment by any13th** [comment]
So go to the manga info tab, choose whatever website you want to download from. 
Next search whatever manga you want to read. *Note 
Then, right click the manga for manga info.(Also to the right of the search bar, click info) 
In the huge white area below, if you see chapters or volumes of pretty much anything, then you can go ahead and download that manga. (If not, then your out of luck and that manga doesn't have volumes or chapters on the specific website)
You have two options to do so, either download specific chapters/volumes of the manga or downloading the whole thing by right clicking the manga again and choosing "Download All". 
I'd suggest downloading just one chapter to test the waters.
The downloaded manga will be moved to the "downloads" folder(lower case d) in the Free Manga Downloader main folder. 
It'll be a ton of separated images.
I suggest viewed them by using Honey View, which allows you to view all the images in a folder like a slideshow. It's especially nice because the image wraps around automatically.
Hope this helps.

**Comment by any13th** [comment]
Also there's the chance you may have downloaded the wrong thing.
Originally I downloaded the files in the code, but I realized that there was no application to launch in them so I took a look around at the site and found this.
https://sf.net/p/fmd/
It's at the top of the code section in this project it says "Forked from https://sf.net/p/fmd/".
I have no clue what forked means in this context, but that link lets you download FMD for real.
Maybe that's the issue.

**Comment by any13th** [comment]
Also here's how to add the other manga websites. 
Click on the Options Tab.
Click on the Websites Tab.
Choose your language(s) and check off whichever websites you want to add. (Note there are two options for English)
Then press apply to add them.
Next time when you search, you should see them as an option, although you will have to download the manga list for that website, but it won't take long.

**Comment by cnavarroestrella** [comment]
WARNING!

Forked means that they took the code from the original unmaintained FMD project and started making changes on their own. The proper FMD now is https://sf.net/p/newfmd/ and not https://sf.net/p/fmd/

Also this works just like Domdomsoft. Go to the "Manga info" tab, select a manga server (you can activate more in options) and type the name.

**Comment by mark3701** [comment]
Здравствуйте! Подскажите, а FMD работает? Ибо у меня все пусто чтобы я не нажимал, а еще постоянно вылазит окно с ошибками
![Снимок экрана 2025-01-10 141654](https://github.com/user-attachments/assets/b98e0ce4-b04e-4978-95b7-91fb125a4476)
![Снимок экрана 2025-01-10 141755](https://github.com/user-attachments/assets/e51f9606-d6bf-4dc9-a2a9-d23c4c6c4d9e)
